### PR TITLE
Closes #15 by allowing multiple arguments in mutation requests. 

### LINKED ISSUE #15: Mutation - Passing in an array of Ints for Mutating Field value
Here's the mutation I'm trying to accomplish:

screen shot 2016-08-27 at 5 22 02 pm

However, I'm getting an error when trying to set up my MutatingField values like so:

screen shot 2016-08-27 at 5 23 02 pm

Is it possible to accomplish this?

### PR DESCRIPTION
**Pull request by vittoriom**
Breaks the API in several places.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,16 +86,16 @@ let imageContent = Fragment(
...
+**GraphQL** also gives us the possibility to have custom enums as argument values. All you have to do is letting your enum implement `ArgumentValue` and you're good to go.
 
 ```swift
 enum customEnum: String, ArgumentValue {
   case This = "this"
   case That = "that"
   
   private var asGraphQLArgument: String {
-    return rawValue // without quotes
+	return rawValue // without quotes
```
Comment (joemcbomb) [inline README.md]: Indentation here?

File: Sources/Mutation/ObjectValue.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+//  ObjectValue.swift
+//  GraphQLicious
+//
+//  Created by Monaco, Vittorio on 30/08/16.
+//  Copyright © 2016 WeltN24. All rights reserved.
+//
+
+import Foundation
+
+public struct ObjectValue: ArgumentValue {
```
Comment (joemcbomb) [inline Sources/Mutation/ObjectValue.swift]: Could we add some documentation here?

File: Sources/Mutation/ObjectValue.swift [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    self.keyValuePairs = keyValuePairs
+  }
+  
+  public var asGraphQLArgument: String {
+    let arguments = keyValuePairs.isEmpty ? "" : "\(keyValuePairs.map {$0.asGraphQLString}.joinWithSeparator(","))"
+    return "{\n\t\t\(arguments)\n}"
+  }
+}
+
+public typealias ObjectKeyValuePair = Argument
```
Comment (joemcbomb) [inline Sources/Mutation/ObjectValue.swift]: and here?

**Comment by joemcbomb** [comment]
Should we update the readme since we break the API?

**Comment by vittoriom** [comment]
Would you rather add a CHANGELOG.md file or just a paragraph in the README.md called "Breaking changes" or something similar?

**Comment by joemcbomb** [comment]
An updated readme should suffice for now. When we do the next release we can add a 'breaking changes' paragraph

File: README.md (hunk 1 above)
Comment (vittoriom) [inline README.md]: done

File: Sources/Mutation/ObjectValue.swift (hunk 2 above)
Comment (vittoriom) [inline Sources/Mutation/ObjectValue.swift]: done

File: Sources/Mutation/ObjectValue.swift (hunk 3 above)
Comment (vittoriom) [inline Sources/Mutation/ObjectValue.swift]: done

**Comment by vittoriom** [comment]
done

**Pull request merged**